src/protocolgame.cpp

```diff
@@ -2449,16 +2449,16 @@ void ProtocolGame::sendCreatureTurn(const Creature* creature, uint32_t stackpos)
 	}
 
 	NetworkMessage msg;
-	msg.addByte(0x6B);
+	msg.addByte(GAME_SERVER_CHANGE_ON_MAP);
 	if (stackpos >= MAX_STACKPOS) {
-		msg.add<uint16_t>(0xFFFF);
+		msg.add<uint16_t>(0xFFFF); // use creature id instead position
 		msg.add<uint32_t>(creature->getID());
 	} else {
 		msg.addPosition(creature->getPosition());
 		msg.addByte(stackpos);
 	}
 
-	msg.add<uint16_t>(0x63);
+	msg.addByte(GAME_SERVER_SEND_CLIENT_CHECK);
```

Comment:
Otclient does not handle the rest of this packet.
it seems to me that this rest is `GAME_SERVER_SEND_CLIENT_CHECK`.

We don't have a turn packet at otclient 😞